src/shaders/pbr.frag

```diff
@@ -221,16 +282,40 @@ void microfacetModel(vec3 specularReflectance,
   float temp = max(4.0 * n_dot_l * n_dot_v, Epsilon);
   vec3 specular = Fresnel *
                   specularGeometricAttenuation(n_dot_l, n_dot_v, roughness) *
-  // normalDistributionGGX is defined in the pbrCommon.glsl
-  // Specular D, normal distribution function (NDF),
-  // also known as ggxDistribution
+                  // normalDistributionGGX is defined in the pbrCommon.glsl
+                  // Specular D, normal distribution function (NDF),
+                  // also known as ggxDistribution
                   normalDistributionGGX(n_dot_h, roughness) / temp;
 
   vec3 tempVec = lightRadiance * n_dot_l;
   diffuseContrib = diffuse * tempVec;
   specularContrib = specular * tempVec;
 }
 
+#if defined(IMAGE_BASED_LIGHTING)
+// c_diff: diffuse color
+// n: normal on shading location in world space
+vec3 computeIBLDiffuse(vec3 c_diff, vec3 n) {
+  // diffuse part = c_diff * irradiance
+  // return c_diff * texture(IrradianceMap, n).rgb * Scales.iblDiffuse;
+  return c_diff * SRGBtoLINEAR(tonemap(texture(IrradianceMap, n))).rgb *
```

Comment:
Wait, doing sRGB->linear conversion _after_ applying a tonemap? That's definitely not right.

Actually, could you just drop the whole `SRGBtoLINEAR` function altogether, as we talked about before? It doesn't do anything at the moment because `MANUAL_SRGB` isn't defined, here it's even used incorrectly and the operation should be done by the hardware anyway. So even once our pipeline becomes sRGB-correct, the function would have no use here, the hardware does that instead.